megalista_dataflow/requirements.txt

```diff
@@ -1,13 +1,12 @@
 apache-beam==2.51.0
-google-ads-megalista
+google-ads==22.1.0
```

Comment:
Should we bump the google-ads-megalista instead?

Reply:
Since we're able to use the official version, I think we reduce the overhead of keeping and supporting another component.